src/lib/components/index/hero.svelte

```diff
@@ -1,28 +1,50 @@
+<script>
+  import PrimaryButton from "../library/primary-button.svelte";
+</script>
+
 <div
-  class="mt-x-large relative w-11/12 md:w-fit mx-auto justify-between text-left md:text-left md:mb-[400px]"
+  class="mt-x-large relative w-11/12 md:w-fit mx-auto justify-between text-left md:mb-[420px]"
 >
-  <h1 class="text-5xl md:text-6xl font-extrabold md:ml-36">
-    CDE <span class="font-light">universe</span>
-  </h1>
-  <div class="mt-4 text-xl md:text-[28px] md:leading-[2.5rem] md:ml-36">
-    June 1 and 2
-  </div>
-  <div
-    class="mt-1 font-medium text-rose text-xl md:text-[28px] md:leading-[2.5rem] md:ml-36"
-  >
-    <div>In-Person, Pebblebed</div>
-    <div>San Francisco, CA</div>
+  <div class="md:ml-36">
+    <h1 class="text-5xl md:text-6xl font-extrabold">
+      CDE <span class="font-light">universe</span>
+    </h1>
+
+    <div class="mt-4 text-xl md:text-[28px] md:leading-[2.5rem]">
+      June 1 and 2
+    </div>
+
+    <div
+      class="mt-1 font-medium text-rose text-xl md:text-[28px] md:leading-[2.5rem]"
+    >
+      <div>In-Person, Pebblebed</div>
+      <div>San Francisco, CA</div>
+    </div>
+
+    <div class="mt-6">
+      <PrimaryButton
+        href="https://www.eventbrite.co.uk/e/cde-universe-2023-cloud-development-conference-by-gitpod-tickets-626052498877"
+        text="Buy ticket"
+      />
+    </div>
   </div>
+
   <img
     src="/images/primary-ellipse.webp"
     alt="CDE Ellipse"
     draggable={false}
     class="hidden md:flex max-w-4xl -z-10 select-none w-[85vw] transfrom scale-[2.12] absolute translate-x-[100%] translate-y-[-180%] rotate-[-6deg] md:transform md:translate-x-[25%] md:translate-y-[-95%] md:scale-[2.10]"
   />
+
   <img
     style="top: -20%; left: 30%; transform: translate(-20%, -50%)"
     class="flex md:hidden absolute w-screen min-w-[65vw] z-[-10] scale-150"
     src="/images/primary-ellipse-mobile.webp"
     alt="ring line"
   />
 </div>
+
+<p class="text-left md:text-center max-w-2xl mx-auto text-xl mt-16">
+  Gitpod's conference for you to learn about cloud-native developer tools and
```

Comment:
Could you left-align the text approximately on the same width as the highlighted topics? (see new [mockup](https://www.figma.com/file/Gm3Fs2yADw6xFrmJkeXk1V/CDE-universe?type=design&node-id=558%3A1170&t=hJ1pYbwwPoP998Sc-1))
Not the best solution but better than the current one :) 


<img width="728" alt="Bildschirmfoto 2023-05-12 um 09 48 43" src="https://github.com/gitpod-io/CDE-Universe/assets/46345125/6023c90b-5037-43b2-bb3b-bdbe91d223bc">

Reply:
this left aligned description text is looking weird tbh 🙈

Reply:
Mh I think it looks better and it's also easier to read. @MichaelAring WDYT? 
see [mockup](https://www.figma.com/file/Gm3Fs2yADw6xFrmJkeXk1V/CDE-universe?type=design&node-id=558%3A1170&t=hJ1pYbwwPoP998Sc-1)

Reply:
resolved according to your suggestions.